### ISSUE TITLE
Auto Update Target List

### ISSUE THREAD

**Issue by burton1982** [salesagility/SuiteCRM#41]
Hello! 

It would be nice if you could implement a feature to auto update the target. Like here: http://www.epicom.com/blog/amandas-blog/auto-update-target-lists-in-sugarcrm

Have a nice day!

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Auto Update Target List](https://trello.com/c/YHOoOFCp/168-auto-update-target-list)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
